runtime/caml/fiber.h

```diff
@@ -168,14 +168,15 @@ struct c_stack_link {
  *   PERFORM captures the current stack in a continuation object it allocates.
  *   The parent stack is then switched to and the handle_effect function for
  *   the parent stack is executed. If no parent stack exists then the
- *   Unhandled exception is raised.
+ *   Effect.Unhandled exception is raised.
  *
  *  Preperform -> REPERFORMTERM
  *   REPERFORMTERM is used to walk up the parent OCaml stacks to execute the
  *   next effect handler installed in the chain. The instruction takes care to
- *   switch back to the continuation stack to raise the Unhandled exception in
- *   in the case no parent is left. Otherwise the instruction switches to the
- *   parent stack and executes the handle_effect function for that parent stack.
+ *   switch back to the continuation stack to raise the Unhandledi_effect
```

Comment:
Typo `Unhandledi_effect` -> `Effect.Unhandled`

Reply:
@Octachron It seems you missed this one. :slightly_smiling_face: